balboa-core/src/main/scala/com/socrata/balboa/metrics/data/impl/CassandraUtil.scala

```diff
@@ -37,9 +38,18 @@ object CassandraUtil extends StrictLogging {
 
     def execute(stmt: Statement): ju.List[Row] = {
       val session = this.newSession
-      val result = session.execute(stmt).all()
+      val result = Try(session.execute(stmt).all())
```

Comment:
It gets caught, then thrown on the call to `.get`. I can add a comment to that effect if you'd like. I considered doing it, but then wasn't sure if it would be obvious to people more used to scala than I.